packages/daemon/src/daemon.js

```diff
@@ -541,7 +541,7 @@ export const main = async (powers, locator, pid, cancel, cancelled) => {
 
   await powers.initializePersistence(locator);
 
-  const requestedWebletPortText = powers.env.ENDO_HTTP_PORT;
+  const requestedWebletPortText = powers.endoHttpPort;
```

Comment:
I look forward to making the daemon’s “services” platform specific to the extent that is possible. That might mean that some power facet methods like `requestOpen` or `open` for weblets might have to get rejected early (not supported) or initialized late depending (requires user intervention to enable, like selecting an HTTP port).